spotify_dl/spotify_dl.py

```diff
@@ -29,22 +30,24 @@ def spotify_dl():
     parser.add_argument('-v', '--version', action='store_true',
                         help='Shows current version of the program')
     parser.add_argument('-o', '--output', type=str, action='store',
-                        nargs='*', help='Specify download directory.')
```

Comment:
`nargs='*'` sends that arguments as a list. I'm assuming that there can only be one output directory at a time for a playlist. so we just need one value. returning list just makes thing complicated. This fixes #44.